### PR TITLE
Add Lib/TruFont.spec

### PR DESCRIPTION
**Pull request by davelab6**
Update Lib/build.sh to use spec file, to fix https://github.com/trufont/trufont/issues/54

### CODE REVIEW

File: Lib/TruFont.spec [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          a.datas,
+          name='TruFont',
+          debug=False,
+          strip=None,
+          upx=True,
+          console=False, 
+		  icon='defconQt/resources/app.ico')
+app = BUNDLE(exe,
+             name='TruFont.app',
+             icon='defconQt/resources/app.ico',
```
Comment (adrientetar) [inline Lib/TruFont.spec]: On OSX you need to use the icns icon, not ico.

File: Lib/build.sh [hunk 2]
```diff
@@ -1 +1,7 @@
-pyinstaller --noconsole --name "TruFont" --onefile -i "defconQt/resources/app.ico"  defconQt/__main__.py
+pyinstaller \
+	--noconsole \
+	--name "TruFont" \
+	--onefile \
+	--icon="defconQt/resources/app.ico" \
+	--osx-bundle-identifier=io.github.trufont \
+	TruFont.spec;
```
Comment (adrientetar) [inline Lib/build.sh]: Does this work with the other platforms? Maybe this file should be changed into a python script.

File: Lib/TruFont.spec (hunk 1 above)
Comment (davelab6) [inline Lib/TruFont.spec]: Thanks!

File: Lib/build.sh (hunk 2 above)
Comment (davelab6) [inline Lib/build.sh]: It won't work on windows; I've reverted this.

**Pull request merged**